python/example/single_cell_detailed_recipe.py

```diff
@@ -105,44 +100,44 @@ def __init__(self, cell, probes):
         self.the_props.set_ion(ion="ca", int_con=5e-5, ext_con=2, rev_pot=132.5)
         self.the_props.catalogue.extend(arbor.allen_catalogue(), "")
 
-    # (6.2) Override the num_cells method
+    # (5.2) Override the num_cells method
     def num_cells(self):
         return 1
 
-    # (6.3) Override the num_targets method
+    # (5.3) Override the num_targets method
     def cell_kind(self, gid):
         return arbor.cell_kind.cable
 
-    # (6.4) Override the cell_description method
+    # (5.4) Override the cell_description method
     def cell_description(self, gid):
         return self.the_cell
 
-    # (6.5) Override the probes method
+    # (5.5) Override the probes method
     def probes(self, gid):
         return self.the_probes
 
-    # (6.6) Override the connections_on method
+    # (5.6) Override the connections_on method
     def connections_on(self, gid):
         return []
 
-    # (6.7) Override the gap_junction_on method
+    # (5.7) Override the gap_junction_on method
     def gap_junction_on(self, gid):
         return []
 
-    # (6.8) Override the event_generators method
+    # (5.8) Override the event_generators method
     def event_generators(self, gid):
         return []
 
-    # (6.9) Overrode the global_properties method
+    # (5.9) Overrode the global_properties method
     def global_properties(self, gid):
         return self.the_props
 
 
 # Instantiate recipe
 # Pass the probe in a list because that it what single_recipe expects.
-recipe = single_recipe(cell, [probe])
+recipe = single_recipe(cell, [arbor.cable_probe_membrane_voltage('"custom_terminal"')])
```

Comment:
> Can't we move the probe list into the class

Not sure what you mean by that.

Reply:
I'll move the probes, but the cell_description should probably stay broken out keep the flow of the code simpler: to keep it easy to separately discuss the cell and recipe. I'll remove the cell from the constructor.

Reply:
That's fine, but might warrant a comment (or sentence in the text) along the lines of:
We put it outside the class to make the cell only once (for efficiency) and up front (for explanation),
but this might be done just as well in `__init__` or in `get_cell_description`. The latter 
will be called multiple times (possibly > `num_cells` even).

Reply:
I've gathered up all the hard requirements to the `recipe` structure into an infobox, and made it more clear that other than meeting those requirements, the example is just an example.